### DISCUSSION TITLE
Cannot display in full screen

### DISCUSSION THREAD

**Discussion by scapking** [zoontek/react-native-edge-to-edge#51] [Q&A]
I have been trying or looking for a component that can be displayed in full screen.

My react-native was created using expo, 

According to the instructions of the document, I configured: /android/app/src/main/res/values/styles.xml

Replacing, light, or, dark, reopening has the corresponding effect.

But it cannot be displayed in full screen,
(beginner)

**Comment by scapking** [comment]
picture

![Screenshot_2024-12-11-14-27-34-85_c3501d829c313dc8c9a74e675ab425f9](https://github.com/user-attachments/assets/4194fd7b-cc2e-480a-804b-e94b03155d58)

![Screenshot_2024-12-11-14-27-20-91_c3501d829c313dc8c9a74e675ab425f9](https://github.com/user-attachments/assets/14411c88-5b4e-437f-a9c5-8e935f7e28eb)

**Comment by scapking** [comment]
`android/app/src/main/res/values/styles.xml`

![Screenshot from 2024-12-11 14-37-35](https://github.com/user-attachments/assets/6eb23aeb-7185-4d1a-8d13-b39cacc966b8)

**Reply by zoontek** [reply]
You have an `android:statusBarColor` set to `#ffffff`

This kind of attributes are automatically removed by the Expo plugin.

**Comment by scapking** [comment]
`package.json`

![Screenshot from 2024-12-11 14-39-11](https://github.com/user-attachments/assets/a673c08d-72ca-4968-a2a7-29396f16d85e)

**Reply by zoontek** [reply]
Don't install is-edge-to-edge, follow what's written in the documentation

**Comment by scapking** [comment]
`index.tsx`

![image](https://github.com/user-attachments/assets/7b8c5c65-74ad-4ad3-b0c6-0b7710599ac8)

**Comment by scapking** [comment]
The documentation or examples are not friendly to beginners

**Comment by zoontek** [comment]
Pretty sure you don't create a build for your app.

Use `npx android`, not `npx start`

**Reply by scapking** [reply]
> Pretty sure you don't create a build for your app.
> 
> Use `npx android`, not `npx start`

My device performance is average, so I use,

`npx expo prebuild`

Then

`cd android`
`./gradlew assembleDebug`



Can you give me a more detailed document?

For example: `npx create-expo-app example`

`npx expo install react-natvie-edge-to-edge` configure a series of steps, if possible,

And the index.tsx code example


You have an android:statusBarColor set to #ffffff

This kind of attributes are automatically removed by the Expo plugin.

This is the default configuration, I did not modify it: android:statusBarColo #ffffff,
（If it is the main cause of the problem, I would try to modify or remove）

**Reply by zoontek** [reply]
Publish your repository.

> Can you give me a more detailed document?

There's a link in the README on "create a build" that links to the Expo documentation.

**Reply by scapking** [reply]
Wait a minute while I create my repository

**Reply by scapking** [reply]
> Publish your repository.
> 
> > Can you give me a more detailed document?
> 
> There's a link in the README on "create a build" that links to the Expo documentation.



Creating a Github repository in a short period of time may be cumbersome.

Package a compressed package, or do not compress it.

Can I send it to you through a web server?

**Reply by scapking** [reply]
> Publish your repository.
> 
> > Can you give me a more detailed document?
> 
> There's a link in the README on "create a build" that links to the Expo documentation.

Are you still there?

**Reply by zoontek** [reply]
@scapking Man. This is free support. Don't be pushy.
No, push the repository on GitHub or I will close this discussion.

**Reply by scapking** [reply]
okey Thanks for your help
